cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -9,14 +9,15 @@
 # - SPLUNK_HEC_TOKEN: The Splunk HEC authentication token
 # - SPLUNK_HEC_URL: The Splunk HEC endpoint URL, e.g. https://ingest.us0.signalfx.com/v1/log
 # - SPLUNK_INGEST_URL: The Splunk ingest URL, e.g. https://ingest.us0.signalfx.com
+# - SPLUNK_NETWORK_INTERFACE: The network interface the agent receivers listen on.
```

Comment:
I think denoting the directionality is helpful since `SPLUNK_NETWORK_INTERFACE` could be ambiguous and may be misinterpreted as an address to reach splunk

```suggestion
# - SPLUNK_LISTEN_INTERFACE: The network interface the agent receivers listen on.
```

Reply:
Thanks, this is better. I will adopt it across.